Doc/library/fcntl.rst

```diff
@@ -184,6 +184,8 @@ The module defines the following functions:
    .. versionchanged:: next
       The GIL is always released during a system call.
       System calls failing with EINTR are automatically retried.
```

Comment:
Same remark, you should update the PR and add a new entry for Python 3.15 (next).